frontend/server/src/main/java/com/amazonaws/ml/mms/util/logging/QLogLayout.java

```diff
@@ -14,27 +14,36 @@
 
 import com.amazonaws.ml.mms.metrics.Dimension;
 import com.amazonaws.ml.mms.metrics.Metric;
+import org.apache.log4j.Logger;
 import org.apache.log4j.PatternLayout;
 import org.apache.log4j.spi.LoggingEvent;
 
 public class QLogLayout extends PatternLayout {
+    private static final Logger logger = Logger.getLogger(QLogLayout.class);
 
     @Override
     public String format(LoggingEvent event) {
         Object eventMessage = event.getMessage();
         if (eventMessage == null) {
             return null;
         }
+        String marketPlace = "NONE";
+        try {
+            marketPlace = System.getenv("REALM");
+        } catch (Exception e) {
```

Comment:
There should not be a expection

Reply:
String marketPlavce = System.getEnv("REALM");
if (marketPlace == null) {
    marketPlace = "NONE";
}